src/pages/registration/RegistrationFormActions.tsx

```diff
@@ -138,6 +139,7 @@ export const RegistrationFormActions = ({
           gridTemplateColumns: { xs: 'auto auto auto', md: '1fr 1fr 1fr' },
           alignItems: 'center',
           gap: '1rem',
+          border: 'none',
```

Comment:
I stedet for å måtte sette `boder: 'none'` alle plasser vi skal ha dette foreslår jeg å legge i theme at vi ikke vil ha border på fieldset:

```
    components: {
      MuiCssBaseline: {
        styleOverrides: {
          fieldset: {
            border: 'none',
          },
        },
      },
```

```suggestion
```